### PR TITLE
More dependencies, move PVT src to scripts

### PR DESCRIPTION
**Pull request by DLu**

### CODE REVIEW

File: face_detector/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,9 @@ find_package(catkin REQUIRED COMPONENTS
   image_geometry
   actionlib
   actionlib_msgs
-  message_generation)
```
Comment (dlaz) [inline face_detector/CMakeLists.txt:19]: Action generation doesn't require rospy, right?

Comment (DLu) [inline face_detector/CMakeLists.txt:19]: Unsure, but you have face_detector_action_client.py

Comment (dlaz) [inline face_detector/CMakeLists.txt:19]: Indeed

File: people_velocity_tracker/package.xml [hunk 2]
```diff
@@ -11,11 +11,17 @@
...
+  <build_depend>kalman_filter</build_depend>
+  <build_depend>roslib</build_depend>
+  <build_depend>rospy</build_depend>
   
   <run_depend>geometry_msgs</run_depend>
   <run_depend>people_msgs</run_depend>
   <run_depend>rospy</run_depend>
   <run_depend>easy_markers</run_depend>
   <run_depend>leg_detector</run_depend>
+  <run_depend>kalman_filter</run_depend>
```
Comment (dlaz) [inline people_velocity_tracker/package.xml:24]: Prerelease failed because it couldn't resolve the kalman_filter key. Where should that be coming from?

Comment (DLu) [inline people_velocity_tracker/package.xml:24]: It's coming through as part of the wu_ros_tools pkg which should be built by later today. I'll ping you when its ready.

**Pull request merged**